### DISCUSSION TITLE
License key

### DISCUSSION THREAD

**Discussion by Russellsoad** [lucky-media/landtamic#14] [General]
Good afternoon. I've got one question I would be really glad if anybody could help me with.
We purchased the key, and chose the option to determine the domain later, can we work with one key locally/dev/and then on the main domain? So the question is will it work correctly on 2-3 domains as we need to activate it on our dev server firstly?

**Comment by arlindmusliu** [comment]
Hey there! We understand the issue you're facing with.
But when it comes to licensing, you need to chat with the folks over at Statamic.

Don't worry though - we're happy to help you out with anything else related to the Starter Kit.

**Reply by Russellsoad** [reply]
Ok, thank you for the quick response

**Reply by Russellsoad** [reply]
Sorry for the missleading. The thing was about the Landtamic license. Could you please help regarding that? 
![image](https://user-images.githubusercontent.com/30286991/234826373-117414fd-09f3-4062-a54c-4d6e54c753c0.png)

**Reply by arlindmusliu** [reply]
That's the thing, Starter Kit licenses are dealt with by the Statamic team. They'd be more than happy to assist you with that :)
